.github/workflows/patch.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       run:
         shell: bash
 
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
Surprised this is needed?  but lgtm

Reply:
yeah messing with it now to see what the deal is 😆